src/js/views/patients/shared/components/due_component.js

```diff
@@ -68,7 +69,7 @@ export default Component.extend({
         defaultText: isCompact ? '' : i18n.defaultText,
         dateFormat: isCompact ? 'SHORT' : 'LONG',
         date: selected,
-        isOverdue: selected ? dayjs(selected).isBefore(dayjs(), 'day') : false,
+        isOverdue: (!isDisabled && selected) ? dayjs(selected).isBefore(dayjs(), 'day') : false,
```

Comment:
This component will have to use the same overdue logic now as the time one

Reply:
Jumped the gun pushing those changes up. Just about wrapped up the due component change.